app/code/community/Elgentos/CodebaseExceptions/Helper/Data.php

```diff
@@ -41,4 +41,35 @@ public function insertException($reportData) {
 
         $this->client->notifyOnError($reportData[0],$backtraces);
     }
+
+    /**
+     * @param string $message
+     */
+    public function sendToAirbrake($message) {
+        $backtraceArray = explode("\n", $message);
+        if (count($message) < 1) {
```

Comment:
It looks like `count("any string")` always returns `1`. I guess you wanted to use `$backtraceArray` here, no? Nevertheless, I don't understand why we need to perform this check at all, even with `$backtraceArray`. Could you please explain?

If this is indeed needed, I'd suggest checking for emptiness (e.g. something like `if (empty($message)) {`). That said, it's not clear to me why we need this :)